tests/rptest/tests/rpk_topic_test.py

```diff
@@ -161,6 +163,10 @@ def test_consume_from_partition(self):
         n_parts = random.randint(3, 100)
         self._rpk.create_topic(topic, partitions=n_parts)
 
+        # Gotta sleep to make sure the topic is replicated and the
+        # consumer doesn't fail.
+        time.sleep(5)
```

Comment:
this doesn't make sense to me. 

@mmaslankaprv why do we have to sleep for 15 seconds.

Reply:
@0x5d use the `wait_until` ducktape helper with predicate to test for the condition you are waiting on.

Reply:
For some reason I don't understand, nope. It's gotta be a sleep and it kills me.

Reply:
please this with wait_until?

Reply:
can you provide some information about why it won't work? if sleep works, then wait_until should work.